github/Organization.py

```diff
@@ -571,33 +571,42 @@ def create_secret(
         secret_name: str,
         unencrypted_value: str,
         visibility: str = "all",
+        secret_type: str = "actions",
         selected_repositories: Opt[list[github.Repository.Repository]] = NotSet,
     ) -> github.OrganizationSecret.OrganizationSecret:
         """
-        :calls: `PUT /orgs/{org}/actions/secrets/{secret_name} <https://docs.github.com/en/rest/actions/secrets#create-or-update-an-organization-secret>`_
+        :calls: `PUT /orgs/{org}/{secret_type}/secrets/{secret_name} <https://docs.github.com/en/rest/actions/secrets#create-or-update-an-organization-secret>`_
         """
```

Comment:
Please add a respective `:param: line for `secret_type` to document the allowed values
```suggestion
        :param secret_type: string "actions" or "dependabot"
        """
```
https://github.com/PyGithub/PyGithub/commit/603896f44541740d72aa441ed60c698760a1393e#diff-dae08970e0b8e7ed63ea3762f066e99270b9262feac1ca1548beb96179d21d42R610